gnomad_qc/v4/create_release/create_release_sites_ht.py

```diff
@@ -267,8 +267,8 @@ def get_config(
             "path": release_sites(data_type=data_type).path,
         },
         "joint_faf": {
-            "ht": get_combined_faf_release().ht(),
-            "path": get_combined_faf_release().path,
+            "ht": get_combined_faf_release(filtered=True).ht(),
```

Comment:
Do we want this as an option, I assume we would always want to use the filtered version so this is probably fine?

Reply:
agreed that we should use the filtered version -- AS_lowqual variants are only present due to the lossless conversion of gVCF>VDS, and they wouldn't be emitted during standard GATK joint calling